.coveragerc

```diff
@@ -2,3 +2,6 @@
 source =
     wandb/
     tests/
+omit =
+    wandb/vendor/*
+    wandb/sdk_py27/*
```

Comment:
I think we do want the merged coverage to include py27

Reply:
I was hoping we could only make calculate py27 coverage when running py27 as it would count against our covered lines in all py3x tests...  I don't think this actually worked though as the percentage didn't go up much.

Reply:
i gave up trying to get creative with partial tox executions and focused on full runs.

coveralls numbers should be merged coverage numbers across py27,py36,py37 and windows